### PR TITLE
#166789886 List all profiles functionality

### PR DESCRIPTION
**Pull request by j4l13n**
#### What does this PR do?
This pull request adds resources that allow users to: 
- add get all profiles method
- add the get all profiles route
- add test of get all method

#### Description of Task to be completed?
A logged in/authenticated user should be able to see the the list and profiles of existing authors. He should however not be able to edit the profiles.

#### How should this be manually tested?
Follow instructions on the `README.md` file attached to this repo to set up the project on your local machine.
Make sure that you have added all environment variables using `example.env`
- Run the application
```
> npm run dev
```

Access the following routes to test

> `/api/profiles` to get all profiles when you have logged in 

Test using Heroku links (click to the link to test):

[Heroku link](https://ah-lobos-backend-swagger.herokuapp.com/#/profile/get_api_profiles)

#### Any background context you want to provide?
- user must be logged in to view authors profiles
#### What are the relevant pivotal tracker stories?
[#166789886](https://www.pivotaltracker.com/story/show/166789886)
#### Screenshots (if appropriate)
image
image

- mockup
image

#### Questions:
- N/A

### CODE REVIEW

**Review by adafia — APPROVED** [review]

**Review by Emile-Nsengimana — APPROVED** [review]

**Review by manzif — APPROVED** [review]

**Comment by emukungu** [comment]
add code climate maintainability to your task for the product.

File: src/controllers/profileController.js [hunk 1]
```diff
@@ -62,6 +62,25 @@ class ProfileManager {
...
+   *
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} all users are returned
+   */
+  static async getAllProfiles(req, res) {
+    try {
+      const findUsers = await Users.findAll({ where: { isVerified: true }, attributes: ['username', 'bio', 'image', 'following'] });
+      res.status(200).json({
+        profile: findUsers
```
Comment (emukungu) [inline src/controllers/profileController.js]: instead of profile, let your return indicate 'users'

File: src/controllers/profileController.js [hunk 2]
```diff
@@ -62,6 +62,25 @@ class ProfileManager {
...
     }
   }
+
+  /**
+   *
+   * @param {Object} req
+   * @param {Object} res
+   * @returns {Object} all users are returned
+   */
+  static async getAllProfiles(req, res) {
```
Comment (emukungu) [inline src/controllers/profileController.js]: change the name to be more descriptive to your task; you are returning a list of users with their profiles

**Review by emukungu — CHANGES_REQUESTED** [review]

**Comment by emukungu** [comment]
where are the mockups?

**Comment by j4l13n** [comment]
> where are the mockups?

I needed your feedback on what I have done, and I am working on I will finish them by tomorrow.

**Comment by emukungu** [comment]
look into the maintainability issue as well. let's change it from C to A

**Comment by j4l13n** [comment]
> look into the maintainability issue as well. let's change it from C to A

I did not get this message sorry, 

> look into the maintainability issue as well. let's change it from C to A

Okay, let me work on it

**Review by Dom58 — APPROVED** [review]

**Pull request merged**